### DISCUSSION TITLE
No access to benefits - Being asked to reverify

### DISCUSSION THREAD

**Discussion by MrsSpindler** [community/community#65224] [GitHub Education]
### Hi everyone,

I reverified my academic status last week, and a few days later received the confirmation email that my benefits are now available. I then requested to upgrade my course organizations, and a few more days later received the confirmation emails that the benefits are available.

However when I try to access my benefits (such as codespaces) I am told I need to reverify.

Is this a bug? This has never been an issue in previous years that I have been using Github Classroom. I understand that there is a waiting period, however it's my understanding that once I receive the email then the waiting period is over. It's also been 4 days since I got those emails, so it can't be the waiting period.

A copy of the email I received:
![image](https://github.com/community/community/assets/31134263/3eda74f3-60b5-4cb8-94d3-1706988a2cdc)

This is what my discount request pages looks like:
![image](https://github.com/community/community/assets/31134263/0ee73fc9-9bc0-400d-adab-7d0a35a859d3)

But when I log into Classroom, I see this banner:
![image](https://github.com/community/community/assets/31134263/025c3338-0d39-45f1-a359-6f24dd7d83c6)

And I get this message when I try to enable codespaces:
![image](https://github.com/community/community/assets/31134263/db3c9f00-b2ea-4052-8265-d46c6d81bf71)

My courses start in a week and I'm worried that I will not be ready for my students. Why is this happening?

**Comment by arafajs** [comment]
- Make sure that you are logged in to the correct GitHub account.
- Make sure that you have accepted the terms of service for the GitHub Student Developer Pack.
- Make sure that you have enabled the GitHub Student Developer Pack benefits in your account settings.
- Try refreshing your browser or clearing your cache and cookies.
- Try using a different browser.
- If you are still having trouble, you can contact GitHub support for help.

**Reply by MrsSpindler** [reply]
Thanks for your response. I am not a student, so the student developer pack isn't the issue. I'm a teacher, and I have gone through this process many times for a number of years. Never before has this happened. It's not a browser issue, nor a cache issue - and I have already created a ticket in Github support.

It appears that I'm not the only one that is having this happen. What do we need to do to get some attention to this issue?

**Comment by kanazuchijin** [comment]
> ### Hi everyone,
> I reverified my academic status last week, and a few days later received the confirmation email that my benefits are now available. I then requested to upgrade my course organizations, and a few more days later received the confirmation emails that the benefits are available.
> 
> However when I try to access my benefits (such as codespaces) I am told I need to reverify.
> 

I am experiencing the exact same problem as you are. Like you, I have used GitHub Classroom and its related programs for years. I have a verified account and even received approval to upgrade my organizations for use in class. Yet, my classes do not reflect that change. The billing plan shows "Free" instead of "Team." I cannot access other benefits such as CodeSpaces. 

I am also asked to reverify, but when I attempt to do so I am given the message, "You were last verified as a teacher on Sep 13, 2022. It is not necessary for you to reverify at this time." My courses started this week and my students cannot access the content.

**Reply by MrsSpindler** [reply]
Yes, that is the exact issue. My organizations are still listed under free, despite received the confirmation that they have received the education discount (and waiting the 2-4 days)  This is very concerning, as it makes it challenging to determine how I should run my course that starts next week.  It's making me reconsidered using Github, which is a shame since up until this issue I've quite enjoyed the experience.
